src/common.h

```diff
@@ -0,0 +1,4 @@
+
+#if defined(DEBUG) || !defined(NDEBUG)
```

Comment:
If I ask a user to compile this with debugging symbols enabled, I'd like to be able to disable this code and avoid a submodule fetch. Couldn't you have a define that says `-DHAVE_READIES` ?

Reply:
We can easily trigger the -DHAVE_READIES through a makefile (by testing for -d deps/readies).
However, I wonder if this really worth the fuss, since:
(a) Most our modules already have submodules (and we'll be introducing RLTools soon),
(b) It's takes just --recursive arg to git clone,
(c) readies enables system-setup.py, which installs required packages to all supported platforms (also available in most modules).
So my guess is that avoiding fetching submodules will become harder as we go.